client/landing/gutenboarding/components/titles.scss

```diff
@@ -0,0 +1,16 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../mixins';
+
+.gutenboarding-title,
+.gutenboarding-subtitle {
+	color: var( --mainColor );
```

Comment:
who declares this one? seems like an overly generic name.

Reply:
Yup, we'll rename. 👍 Thanks :-)

https://github.com/Automattic/wp-calypso/issues/40085
https://github.com/Automattic/wp-calypso/pull/39972#discussion_r391119207

Reply:
It probably is! That's from https://github.com/Automattic/wp-calypso/pull/39928.

I'll add an issue to use a more specific variable for the color.

Reply:
For the record, the variables are here:

https://github.com/Automattic/wp-calypso/blob/cdf2cd28a91c272a268d2d6b774b9cbe9ca9dc56/client/landing/gutenboarding/onboarding-block/colors.scss